docs/topics/multiplatform/multiplatform-run-tests.md

```diff
@@ -1,67 +1,375 @@
+        val name = System.getProperty("java.vm.name") ?: "Android"
+    
+        val version = System.getProperty("java.version")
+    
+        return CurrentRuntime(name, version)
+    }
+    ```
+
+3 Use the IDE's suggestions to create the `androidUnitTest/kotlin` directory:
+
+![Creating Android test directory](create-android-test-dir.png){width=300}
+
+4. Add the `AndroidRuntimeTest.kt` file to the project:
+
+    ```kotlin
+    import kotlin.test.Test
+    import kotlin.test.assertContains
+    import kotlin.test.assertEquals
+    
+    class AndroidRuntimeTest {
+        @Test
+        fun shouldDetectAndroid() {
+            val runtime = determineCurrentRuntime()
+            assertContains(runtime.name, "OpenJDK")
+            assertEquals(runtime.version, "17.0")
+        }
+    }
+    ```
+
+It may seem strange that an Android-specific test is run on a local JVM. This is because these tests run as Local Unit
+Tests on the current machine. As described in
+the [Android Studio documentation](https://developer.android.com/studio/test/test-in-android-studio), these tests are
+distinct from Instrumented Tests, which run on a device or within an emulator.
+
+The **Kotlin Multiplatform App** template project is not configured to support these tests by default. However, it's
+possible to add additional dependencies and folders. You can check out this [Touchlab guide](https://touchlab.co/understanding-and-configuring-your-kmm-test-suite/)
+to add support for Instrumented Tests.
+
+#### For iOS
+
+1. In `iosMain`, create a new `org.kmp.testing/IOSRuntimeTest.kt` file.
+2. Add the actual implementation of the expected function:
+
+    ```kotlin
+    import kotlin.native.Platform
+    
+    actual fun determineCurrentRuntime(): CurrentRuntime {
+        val name = Platform.osFamily.name.lowercase()
+        return CurrentRuntime(name, null)
+    }
+    ```
+
+3. In the same way as for Android, create a folder structure for your iOS tests:
+
+   ![Creating iOS test directory](create-ios-test-dir.png){width=300}
+
+4. Add the `IOSRuntimeTest.kt` file to the project:
+
+    ```kotlin
+    import kotlin.test.Test
+    import kotlin.test.assertEquals
+    
+    class IOSRuntimeTest {
+        @Test
+        fun shouldDetectOS() {
+            val runtime = determineCurrentRuntime()
+            assertEquals(runtime.name, "ios")
+            assertEquals(runtime.version, "unknown")
+        }
+    }
+    ```
+
+## Running multiple tests and reading reports
+
+At this stage, you have the code for common, Android, and iOS implementations, as well as their tests. You should get a
+similar directory structure in your project:
+
+![Whole project structure](code-and-test-structure.png){width=300}
+
+You can run individual tests from the context menu or use the shortcut. One more option is to use Gradle tasks. For
+example, if you run the `allTests` task, every test in your project will be run with a corresponding test runner.
```

Comment:
```suggestion
Instead of running tests individually, you can run multiple tests using Gradle tasks. For
example, if you run the `allTests` Gradle task, every test in your project will be run with a corresponding test runner.
```
I think it would be useful to describe or provide a screenshot where a user can find the `allTests` Gradle task in the IDE.

Reply:
I'm not sure we want to promote one option over the other, just adding one more way to run tests